ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -952,7 +952,7 @@ private[platform] object JdbcLedgerDao {
       validate = false,
       metrics,
       lfValueTranslationCache,
-      jdbcAsyncCommits = jdbcAsyncCommits,
+      jdbcAsyncCommits = jdbcAsyncCommits && dbType.supportsParallelWrites,
```

Comment:
The canton fork file refers to the dbType.supportsAsyncCommit flag where that flag already exists.